### PR TITLE
Update SCSS-Lint to 0.47

### PR DESCRIPTION
**Pull request by tysongach**

### CODE REVIEW

File: .scss-lint.yml [hunk 1]
```diff
@@ -109,6 +109,10 @@ linters:
   PlaceholderInExtend:
     enabled: true
 
+  PrivateNamingConvention:
```
Comment (whmii) [inline .scss-lint.yml]: I assume this just allows there to be `_` prefixed objects?

Comment (tysongach) [inline .scss-lint.yml]: Yes and no. It check for that prefix, but only allows it in the _same file_.

Comment (tysongach) [inline .scss-lint.yml]: This is why I didn't enable it.

Comment (whmii) [inline .scss-lint.yml]: OIC ¯\_(ツ)_/¯

**Comment by whmii** [comment]
:ship:

**Pull request merged**